backend/config/kafkaConfig.ts

```diff
@@ -0,0 +1,11 @@
+import { Kafka } from "kafkajs";
+import { config as dotenvConfig } from "dotenv";
+
+dotenvConfig({ path: __dirname + "/../../.env" });
+
+const kafkaBrokersList = process.env.KAFKA_BROKERS_LIST || "[]";
```

Comment:
Since Kafka is one of the technologies neither of us knows about, we could "use and abuse" of commentaries for instance:
What does this "kafkaBrokerList"? Can you make a brief description in a commentary?